ui/components/app/IPAddress/index.tsx

```diff
@@ -1,31 +1,46 @@
-import { Tooltip, TooltipContent, TooltipTrigger } from '@/components/ui/tooltip';
+import {
+  Tooltip,
+  TooltipContent,
+  TooltipTrigger,
+} from '@/components/ui/tooltip';
 import { CountryFlag, Props as CountryFlagProps } from '../CountryFlag';
 
-
 type Props = Omit<CountryFlagProps, 'isoCode'> & {
-    address: string;
-    isoCode?: string;
-    lat?: string;
-    long?: string;
-}
+  address: string;
+  isoCode?: string;
+  lat?: string;
+  long?: string;
+};
+
+const LatLongContainer = ({
+  lat,
+  long,
+  isoCode,
+  children,
+  ...props
+}: Omit<Props, 'address'> & { children: React.ReactNode }) => (
```

Comment:
`children`, properly typed, is Optional:
```suggestion
}: Omit<Props, 'address'> & { children?: React.ReactNode }) => (
```